Stronger Headlines/StrongerHeadlines.js

```diff
@@ -0,0 +1,23 @@
+tau.mashups
+    .addDependency('tp3/mashups/topmenu')
```

Comment:
Looks like no dependencies needed. Please, remove them.

Reply:
done

Reply:
and `.addDependency('tau/core/bus.reg')` ?

Reply:
not needed, this mashup only applies global css rules.

Reply:
removed

Reply:
Thank you. Merging...